terasoluna-gfw-functionaltest-web/src/main/webapp/WEB-INF/web.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <web-app xmlns="http://java.sun.com/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_3_0.xsd"
-  version="3.0">
+  xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_4_0.xsd"
+  version="4.0">
```

Comment:
- [ ] デフォルト設定はブランクプロジェクトに揃えておくと良いです。
また、3.xと4.xの切り替えをどのような方式で行う想定かを教えてください。

Reply:
In the blank project, we changed to use Servlet 3.0 schema, so we reverted the schema modification.
https://github.com/terasolunaorg/terasoluna-gfw-web-blank/pull/455#discussion_r552296028

https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/905/commits/025840793154fa4abb743e70288d5f8e9467c385

Reply:
Have you considered this?

> Also, please tell us what kind of method you expect to switch between 3.x and 4.x.

Reply:
Since it is impossible to switch namespaces with a single web.xml, I was considering preparing a web.xml for 3.x and a web.xml for 4.x for switching.
Since the referenced xml can be specified using maven-war-plugin, we were going to control it using profiles.
https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#webXml

However, based on the fact that terasoluna:5.6.x did not support it, we judged that there was little need to update to Web-apps 4.x, and did not take the above action.

Reply:
When we build and test Servlet 4.0, we decided to rewrite it to dynamic web module 4.0 just before building.
In addition, since the blank project specifies dynamic web module 3.0 for backward compatibility, terasoluna-gfw-functionaltest will be rewritten in the same way.